src/workerd/server/workerd-api.c++

```diff
@@ -705,9 +706,9 @@ static v8::Local<v8::Value> createBindingValue(JsgWorkerdIsolate::Lock& lock,
         // parsed in one context (env) and unparsed in another (process.env).
 
         if (value->IsString()) {
-          lock.setEnvField(lock.str(global.name), jsg::JsValue(value));
+          lock.setProcessEnvField(lock.str(global.name), jsg::JsValue(value));
```

Comment:
This is getting confusing. process comes from node.js process, but it's extremely hard for a non-nodejs contributor to understand what the "process" prefix is coming from. I recommend picking a more verbose function name like: "setNodeProcessEnvField"

Reply:
eh, I'd rather not make the name longer. I think `ProcessEnv` is fairly descriptive that it applies to `process.env`